src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -2276,6 +2276,171 @@ class StubGenerator: public StubCodeGenerator {
     StubRoutines::_arrayof_jint_fill = generate_fill(T_INT, true, "arrayof_jint_fill");
   }
 
+  void generate_aes_loadkeys(const Register &key, VectorRegister *working_vregs, int reg_number) {
+    const int step = 16;
+    for (int i = 0; i < reg_number; i++) {
+      __ vle32_v(working_vregs[i], key);
+      __ vrev8_v(working_vregs[i], working_vregs[i]);
+      __ addi(key, key, step);
+    }
+  }
+
+  void generate_aes_encrypt(const VectorRegister &res, VectorRegister *working_vregs, int reg_number) {
+    assert(reg_number <= 15, "reg_number should be less than or equal to working_vregs size")
```

Comment:
This lacks `;` at the end of statement, which is causing the GHA failure.

Reply:
Fixed, thanks!